### PR TITLE
Improve expandable table

### PR DESCRIPTION
**Pull request by hkvstore**
1. Add selector '.expandable-body' when selecting the expandable body,
2. Add class "expandable-table-caret" to the caret icon,
3. Update the example in pages\tables\simple.html.

### CODE REVIEW

**Comment by danny007in** [comment]
do not add dist/compiled
check https://github.com/ColorlibHQ/AdminLTE#contributing

**Comment by danny007in** [comment]
explane whats use of this PR in visually or by code

File: build/scss/_table.scss [hunk 1]
```diff
@@ -82,11 +82,11 @@
 [data-widget="expandable-table"] {
   cursor: pointer;
 
-  i {
+  i.expandable-table-caret {
     transition: transform $transition-speed linear;
   }
   &[aria-expanded="true"] {
-    td > i {
+    td i.expandable-table-caret {
```
Comment (danny007in) [inline build/scss/_table.scss:89]: issue may be for child `td > i`

File: pages/tables/simple.html [hunk 2]
```diff
@@ -1446,7 +1446,7 @@ <h3 class="card-title">Expandable Table Tree</h3>
                     </tr>
                     <tr data-widget="expandable-table" aria-expanded="true">
                       <td>
-                        <i class="fas fa-caret-right fa-fw"></i>
+                        <i class="expandable-table-caret fas fa-caret-right fa-fw"></i>
```
Comment (danny007in) [inline pages/tables/simple.html:1449]: y extra `expandable-table-caret`
and 
plz explain whats use of this PR

**Review by danny007in — CHANGES_REQUESTED** [review]

File: build/scss/_table.scss (hunk 1 above)
Comment (hkvstore) [reply build/scss/_table.scss:89]: The icon may not be directly under ``td``.

File: pages/tables/simple.html (hunk 2 above)
Comment (hkvstore) [reply pages/tables/simple.html:1449]: There may be other icons in the expandable row, they will also be affected by the CSS if just ``td > i`` or ``td i`` is used.

File: build/scss/_table.scss (hunk 1 above)
Comment (danny007in) [reply build/scss/_table.scss:89]: is there
any conflict by 
changing
`td i.expandable-table-caret`
to
`td > i.expandable-table-caret`

Comment (hkvstore) [reply build/scss/_table.scss:89]: > The icon may not be directly

File: build/js/ExpandableTable.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ const EVENT_EXPANDED = `expanded${EVENT_KEY}`
 const EVENT_COLLAPSED = `collapsed${EVENT_KEY}`
 
 const SELECTOR_TABLE = '.expandable-table'
+const SELECTOR_BODY = '.expandable-body'
```
[the remark this answers is not in the document]
Comment (hkvstore) [reply build/js/ExpandableTable.js]: ee740e97e49028405478376cad8e4e27ffdfe06a

File: build/scss/_table.scss (hunk 1 above)
Comment (danny007in) [reply build/scss/_table.scss:89]: add extra example in html for

- The icon may not be directly under td.

Comment (hkvstore) [reply build/scss/_table.scss:89]: daa448d4df99a99680083f9c5e6591f1add96914

**Review by danny007in — APPROVED** [review]

**Comment by danny007in** [comment]
@hkvstore i have approved it lets, wait for any other's to take closer review 👍🏼

**Comment by REJack** [comment]
I will check it later

File: pages/tables/simple.html [hunk 4]
```diff
@@ -1480,8 +1480,32 @@ <h3 class="card-title">Expandable Table Tree</h3>
                                   </div>
                                 </td>
                               </tr>
-                              <tr>
-                                <td>219-2</td>
+                              <tr data-widget="expandable-table" aria-expanded="false">
+                                <td>
+                                  <button type="button" class="btn btn-primary p-0">
```
Comment (REJack) [inline pages/tables/simple.html:1485]: Is this intended to be a button?

Comment (danny007in) [reply pages/tables/simple.html:1485]: Yes, @hkvstore 's concept is that only we can add custom button or any element.
And sorry for ignoring that example we needs to add separate example for that.

Comment (danny007in) [reply pages/tables/simple.html:1485]: Also only custom add class gets target not all icons in row

Comment (danny007in) [reply pages/tables/simple.html:1485]: @hkvstore can u add completely separate example below, instead of editing

Comment (REJack) [reply pages/tables/simple.html:1485]: there is a example above as a simple icon ^^ but then it's OK :D I will merge it, it works like it should :P

**Pull request merged**